### PR TITLE
Plops 499 spike to cache docker stages

### PR DESCRIPTION
**Pull request by dannystaple**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,7 @@
 .idea
 .vscode
 
+node_modules
+# Environment files
 .tests/.env
+.env
```
Comment (mikemonkers) [inline .gitignore:8]: Why add .env?

File: .github/Dockerfile [hunk 2]
```diff
@@ -1,3 +1,8 @@
-from ubuntu:latest
+FROM ubuntu:20.04 AS base
```
Comment (mikemonkers) [inline .github/Dockerfile:1]: Why 20.04 here and 20.10 below?

File: Docker_Image_Cache/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM docker:latest
+FROM docker:20.10
```
Comment (mikemonkers) [inline Docker_Image_Cache/Dockerfile:1]: I'm torn here between making the linter happy and using an exception to let it use 'latest'

File: Docker_Image_GetTag/Dockerfile [hunk 4]
```diff
@@ -1,4 +1,4 @@
-FROM docker:latest
+FROM docker:20.10
```
Comment (mikemonkers) [inline Docker_Image_GetTag/Dockerfile:1]: I'm torn here between making the linter happy and using an exception to let it use 'latest'

File: README.md [hunk 5]
```diff
@@ -12,27 +12,25 @@ Repository of Atom shared GitHub actions
 
 [Git Normalise Tag](Git_Normalise_Tag/README.md)
 
+## Configuration
+
+Copy 'tests/.env_example' to '.env' and edit GITHUB_TOKEN to your github
```
Comment (mikemonkers) [inline README.md:17]: Why are we moving the .env to the route directory. It should stay in the tests directory since it only relates to tests

Comment (mikemonkers) [reply README.md:17]: The script expects you to be in the tests directory to run it. The docs should probably show this.

**Review by mikemonkers — CHANGES_REQUESTED** [review]

File: .github/Dockerfile (hunk 2 above)
Comment (dannystaple) [reply .github/Dockerfile:1]: These are ubuntu versions, the others are Docker image versions. Not quite the same tags.

File: .gitignore (hunk 1 above)
Comment (dannystaple) [reply .gitignore]: Whoops - accidentally created that and removed it, but left the ignore in. Removed.

File: README.md (hunk 5 above)
Comment (dannystaple) [reply README.md]: You are quite right - I copied this text from somewhere else, didn't fix it up.

File: Docker_Image_Cache/Dockerfile (hunk 3 above)
Comment (dannystaple) [reply Docker_Image_Cache/Dockerfile:1]: My easy resolution for the same thought was that it's probably better that it won't break unexpectedly with an update. Plus we can set dependabot on it when we get there, hopefully that will deal with docker containers as it does other deps.

File: .gitignore (hunk 1 above)
Comment (mikemonkers) [reply .gitignore]: Oh yeah

File: .github/Dockerfile (hunk 2 above)
Comment (mikemonkers) [reply .github/Dockerfile:1]: Oh yeah

**Review by mikemonkers — APPROVED** [review]

**Pull request merged**